### ISSUE TITLE
[FEAT] Anonymous Input Form

### ISSUE THREAD

**Issue by chaejunlee** [pinkishincoloragain/whoami2#17]
### **Feature**

Feature for developing anonymous name form.
This issue is made to 

1. develop fitting layout
2. features that provide great UX for anonymous writers

### **What should be done**

- [ ] Anonymous input form
- [ ] Create random anonymous user name
- [ ] Way to store data from anonymous user
- [ ] TBA

### **Additional context**

> > ### How to add the "Anonymous" option of "Name or Nickname (이름 & 별명)
> >
> > - I don't think that the provided design (grid) goes well with the flow
> >
> > - Therefore, I went with the previous design (flex or stack)
> >
> > - What do you think? @pinkishincoloragain 
> 
> That makes sense. I think this feature should be developed. 
> @chaejunlee  Can you make new issue? We will discuss..
> 
> - How to create anonymous input form
> - How to create random anonymous user name
> - How to store data form anonymous user
> - ...
> 
> This will involve designer's opinion also.

<img width="1046" alt="Screen Shot 2022-12-18 at 21 54 49" src="https://user-images.githubusercontent.com/39547381/208299538-af26858a-7929-4dd7-af73-85e67814427b.png">

**Comment by pinkishincoloragain** [comment]
## Brainstorming

We will provide two options to users which are 
- user input by himself (사용자 별명 설정)
- we provide options (자동 별명 설정)

So I did brainstorming about the anonymous input feature

<img width="1254" alt="Screen Shot 2022-12-21 at 2 05 15" src="https://user-images.githubusercontent.com/39547381/208724457-9ac83504-4203-4520-91a8-fd1dee9d197d.png">

These are the options I thought. 


## So.. 
Which featrue do you think would be the best fit under current circumstances? @chaejunlee 

I personally want to try MBTI option anonymous feature since it might be fire for female users. :) 

Also, I am open to any other options.. or possibilities which could be applied to current situation!

**Comment by pinkishincoloragain** [comment]
## This feature needs to be done prior to other issues
I think this issue need to be finished prior to implementing the DB sync since individual uid is needed to create letters document. 

## So.. 
Could you help me creating new anonymous input form? 
I think that the idea you proposed - creating user input name like
```js
`${MBTI}-${DINOSAUR}`
```
woule be awesome.

How do you think? @chaejunlee
